lib/rules/selector-attribute-name-disallowed-list/__tests__/index.mjs

```diff
@@ -1,4 +1,5 @@
 import rule from '../index.mjs';
+import { stripIndent } from 'common-tags';
 const { messages, ruleName } = rule;
```

Comment:
```suggestion
import { stripIndent } from 'common-tags';

import rule from '../index.mjs';
const { messages, ruleName } = rule;
```

@ryo-manba We split 3rd party and internal module imports.
3rd party modules come first and are separated by an empty line before listing the internal module imports.